sdk/data/aztables/internal/zz_service_client.go

```diff
@@ -53,19 +50,19 @@ func (client *ServiceClient) GetProperties(ctx context.Context, restype Enum5, c
 }
 
 // getPropertiesCreateRequest creates the GetProperties request.
-func (client *ServiceClient) getPropertiesCreateRequest(ctx context.Context, restype Enum5, comp Enum6, options *ServiceClientGetPropertiesOptions) (*policy.Request, error) {
+func (client *ServiceClient) getPropertiesCreateRequest(ctx context.Context, options *ServiceClientGetPropertiesOptions) (*policy.Request, error) {
 	req, err := runtime.NewRequest(ctx, http.MethodGet, client.endpoint)
 	if err != nil {
 		return nil, err
 	}
 	reqQP := req.Raw().URL.Query()
-	reqQP.Set("restype", string(restype))
-	reqQP.Set("comp", string(comp))
+	reqQP.Set("restype", "service")
+	reqQP.Set("comp", "properties")
 	if options != nil && options.Timeout != nil {
 		reqQP.Set("timeout", strconv.FormatInt(int64(*options.Timeout), 10))
 	}
 	req.Raw().URL.RawQuery = reqQP.Encode()
-	req.Raw().Header["x-ms-version"] = []string{string(client.version)}
+	req.Raw().Header["x-ms-version"] = []string{"2019-02-02"}
```

Comment:
Why do we use a constant here rather than client.version?

Reply:
Historically, autorest treated enums with a single value as a constant.  This default behavior was later changed due to treating enums as extensible by default if not specified.  I set the flag to enable the old behavior which is what we want as these values are constant and not user-configurable.